client/src/utils/indexs.js

```diff
@@ -52,10 +56,11 @@ export const redirectToLocal = (url) => {
 }
 
 export const crontabToText = (crontab) => {
+  const { t } = useTranslation();
```

Comment:
:red_circle:  Never ever use hooks in functions that are not hook. In this case it's being called in the jsx and you cannot predict the first rule of hook.

Reply:
I think I understand what I did wrong. I changed it to pass `t` along when calling the func.